eve/main.yml

```diff
@@ -84,6 +84,48 @@ stages:
         env:
           ANSIBLE_INVENTORY: eve/tests/single-node/inventory/hosts
 
+  multi-node-test:
+    worker:
+      type: multi_vm
+      image: CentOS 7 (PVHVM)
+      flavor: general1-4
+      path: eve/workers/centos7_terraform
+    steps:
+    - Git: *git_pull
+    - ShellCommand:
+        name: "Init terraform"
+        command: "terraform init"
+        workdir: build/contrib/terraform/openstack
+        env:
+          PATH: "$PATH:/usr/local/bin"
+    - ShellCommand:
+        name: "Spawn openstack virtual infra"
+        command: 'terraform apply -auto-approve'
+        workdir: build/contrib/terraform/openstack
+        env: &terraform_spawn_1
+          PATH: "$PATH:/usr/local/bin"
+          OS_AUTH_URL: "%(secret:scality_cloud_auth_url)s"
+          OS_REGION_NAME: "%(secret:scality_cloud_region)s"
+          OS_USERNAME: "%(secret:scality_cloud_username)s"
+          OS_PASSWORD: "%(secret:scality_cloud_password)s"
+          OS_TENANT_NAME: "%(secret:scality_cloud_tenant_name)s"
+          TF_VAR_name_prefix: "%(prop:multivm_prefix)s"
+    - ShellCommand:
+        name: "Destroy openstack virtual infra"
+        command: 'terraform destroy -auto-approve'
+        workdir: build/contrib/terraform/openstack
+        alwaysRun: true
+        env: *terraform_spawn_1
+    - ShellCommand:
```

Comment:
Why twice?

Reply:
Because terraform could fail to delete...(timeout and such) but often running terraform multiple times solve the issue.
I added a bash loop for this

Reply:
Ugh :man_facepalming: OK, I guess.